templates/corporate/upgrade.html

```diff
@@ -101,44 +132,41 @@ <h3>{{ _("License management") }}</h3>
 
                                 <div id="license-automatic-section">
                                     <p>
-                                        {% if free_trial_days %}
-                                        After the Free Trial, you&rsquo;ll be charged
+                                        {% if realm_on_free_trial %}
+                                        You&rsquo;ll initially be charged
                                         <b>$<span id="charged_amount"></span></b> for <b>{{ seat_count }}</b>
-                                        users (or more if you later add more users).<br>
-
-                                        We'll automatically charge you for additional licenses as users
-                                        are added, and remove licenses not in use at the end of each billing
-                                        period.
+                                        users (or more if you later add more users) at the end of free trial.<br>
```

Comment:
Shouldn't this say `end of the free trial` or `end of your free trial` to be correct? Not completely sure, perhaps someone else can confirm

Reply:
Okay. Pasted this string in Grammarly and it autocorrected "end of free" to "end of the free". I have updated the commit.